<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScaleSetupDialog</class>
 <widget class="QDialog" name="ScaleSetupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>484</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Setup Scales</string>
  </property>
  <widget class="QLabel" name="textLabel1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Device:</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="text">
    <string>Baud</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Data Bits</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Parity</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="baudCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>75</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>110</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>134</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>150</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>300</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4800</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>38400</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>230400</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="dataBitCombo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="parityCombo">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>None</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Odd</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Even</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="stopBitCombo">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="browseButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>50</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="devicePathEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>220</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_3_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop Bits</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="startCharCombo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>440</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Character</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2_2_2_2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>End Character</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>411</x>
     <y>440</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel1_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>360</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Format String</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="formatStringEdit">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>400</y>
     <width>180</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="endCharCombo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>400</y>
     <width>140</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="teleWeightCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TeleWeight</string>
   </property>
  </widget>
  <widget class="Q3ButtonGroup" name="retrievalModeGroup">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>180</width>
     <height>90</height>
    </rect>
   </property>
   <property name="title">
    <string>Retrieval Mode</string>
   </property>
   <widget class="QRadioButton" name="continuousRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Continuous</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="demandRadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Demand</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEditMotionChar">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>320</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>M</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelMotionChar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Motion Character: </string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">dao/setupdao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
